<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutCfg</class>
 <widget class="QWidget" name="ShortcutCfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="lstKeys">
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">Command</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Key</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Global</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblKey">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QKeyButton" name="edtKey" native="true"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>&amp;Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkGlobal">
       <property name="text">
        <string>Global</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QKeyButton</class>
   <extends>QWidget</extends>
   <header>simgui/qkeybutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
